### ISSUE TITLE
delta printers / negative offsets

### ISSUE THREAD

**Issue by tmbdev** [iFallUpHill/flow-calculator#22]
It's hard to generate G-code for delta printers with the current system. It would be nice to at least have the option of having negative offsets to fit a square test pattern into the center of a delta printer.

Ideal, of course, would be adding a circular/polar coordinate pattern for round print beds, with flow rates corresponding to different angles and temperatures corresponding to different radii.

**Comment by fefedefe-frog** [comment]
yes it would be really handy, for now I'm using standard flow values ​​taken from the internet for the hotend  I have on my delta, but I'd like to have more accurate and precise ones for each type of filament I use

**Comment by mthqwork** [comment]
+1, it'd be great to have that.
